docs/jwst/extract_1d/description.rst

```diff
@@ -450,6 +450,38 @@ where `flux` is the extracted spectral data, and the data are from channel 4 for
 
 Extraction for NIRISS SOSS Data
 -------------------------------
-For NIRISS SOSS data, the two spectral orders overlap slightly, so a specialized extraction
-algorithm known as ATOCA (Algorithm to Treat Order ContAmination) is used...
-Link paper 
+For NIRISS SOSS data, spectral orders 1 and 2 overlap slightly at longer wavelengths so a specialized extraction
+algorithm known as ATOCA (Algorithm to Treat Order ContAmination, Darveau-Bernier et al., 2022) is made available in
+the ``extract_1d`` step. This routine constructs a linear model of each pixel on the detector and treats the underlying
```

Comment:
```suggestion
algorithm known as ATOCA (Algorithm to Treat Order ContAmination, Darveau-Bernier et al., 2022) is used. This routine constructs a linear model of each pixel on the detector and treats the underlying
```

Reply:
This is fine. I said "made available" since it can be turned off but since it is the default, let's stick with "used".